modules/ROOT/pages/ai-gemini.adoc

```diff
@@ -0,0 +1,224 @@
+= Google Gemini integration guide
+:pluginname: AI Assistant
+:plugincode: ai
+:plugincategory: premium
+:navtitle: Google Gemini integration guide
+:description_short: {pluginname} with Google Gemini
+:description: A guide for integrating {pluginname} plugin using Google Gemini.
+:keywords: example, demo, custom, plugin, ai, assistant, guide, google, gemini
+
+include::partial$misc/admon-ai-pricing.adoc[]
+
+== Introduction
+
+This guide provides instructions for integrating the {pluginname} plugin using Google Gemini in {productname}.
+
+There are two version of the Google Gemini API: Google AI Studio and Google Cloud - Vertex AI. The Google AI Studio API is ideal for quickly integrating the Google Gemini models into an application with the minimal setup required. The Google Cloud - Vertex AI API is more powerful and flexible than Google AI Studio, but requires more setup to integrate the Google Gemini models into an application.
+
+Vertex AI provides a `systemInstruction` field in the request body that allows controlling the output of the AI model. Google AI Studio does not currently provide this field, so the system messages are provided as part of the user prompt.
+
+TIP: Google AI Studio and Vertex AI can both be used to generate string and streaming responses.
+
+To learn more about the difference between string and streaming responses, see xref:ai.adoc#the-respondwith-object[The `respondWith` object] on the plugin page.
+
```

Comment:
We should include a note on each page which recommends setting up a proxy service for authentication and hiding keys, and display it before any examples are shown.

Reply:
@MitchC1999 We already have a comment in the code examples above `const api_key` that states this. Do you want to move that to an admonition outside the code?

Reply:
Yea a note outside the code block would be more visible

Reply:
Added admonition: [93c193f](https://github.com/tinymce/tinymce-docs/pull/3250/commits/93c193f5079db62b3f83b7354ac6b6041e620bc4)

If you are happy with that, I will also add it to OpenAI and Bedrock pages.